go/worker/common/committee/node.go

```diff
@@ -621,6 +658,7 @@ func (n *Node) worker() {
 		)
 		return
 	}
+	atomic.StoreUint32(&n.hostedRuntimeProvisioned, 1)
```

Comment:
The above doesn't block until successful provisioning, so this needs to be deferred until a "started" event is received from the runtime (in `handleRuntimeHostEvent`).

Reply:
Updated